src/Attributes/Field.php

```diff
@@ -262,6 +271,7 @@ public static function create(
             $new->phpType = $phpType;
         }
         $new->typeMap = null;
+        $new->isNullable = $isNullable;
```

Comment:
This is somehow a fallback to prevent failure when `Field` is created from the within serialization...

This basically says that there are places where `null` could be possible, but as this value is not set via `::create` the not null value is enforced.

Reply:
I don't follow.  You're saying that virtual Fields (the ones created during the serialization process) are necessarily non-nullable?  I have no idea off hand if that logically holds in all cases.  Can you verify?

Reply:
In other words I am trying to say that virtual Fields are _currently_ non-nullable, as I currently do not see the way how to (or from where) to initialize this setting. So any idea?

Reply:
If virtual fields are not nullable, and there's no expected use case for them being nullable, then we can skip the parameter and just force them to be not-nullable.  If we do find a use for nullable virtual fields in the future, it's easier to re-add that than to remove it.